python/lsst/tmpl/_ExampleThree.py

```diff
@@ -1,39 +1,36 @@
-# 
-# LSST Data Management System
 #
-# Copyright 2008-2017  AURA/LSST.
-# 
-# This product includes software developed by the
-# LSST Project (http://www.lsst.org/).
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
 #
 # This program is free software: you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation, either version 3 of the License, or
 # (at your option) any later version.
-# 
+#
 # This program is distributed in the hope that it will be useful,
 # but WITHOUT ANY WARRANTY; without even the implied warranty of
 # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
 # GNU General Public License for more details.
-# 
-# You should have received a copy of the LSST License Statement and 
-# the GNU General Public License along with this program.  If not, 
-# see <https://www.lsstcorp.org/LegalNotices/>.
 #
-
-from __future__ import absolute_import
-import numpy as np
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
 
 from lsst.utils import TemplateMeta
-from .exampleThree import ExampleThreeF, ExampleThreeD
 
-__all__ = [] # import for side effects
+from ._tmpl import ExampleThreeF, ExampleThreeD
```

Comment:
I realize this was in the original, but `ExampleThreeF` is not declared above.  (Also applies to the copy of this example in the dev guide.)

Reply:
Switched to `ExampleThreeI` throughout this file.  I'll do a big round of copy-paste from the example repo to the dev guide after verifying that it builds (already done) and imports (not).